pkg/meta/dump.go

```diff
@@ -260,25 +260,24 @@ func (dm *DumpedMeta) writeJsonWithOutTree(w io.Writer) (*bufio.Writer, error) {
 	return bw, nil
 }
 
-func dumpAttr(a *Attr) *DumpedAttr {
```

Comment:
d could be nil?

Reply:
Yes, the caller should make sure that it's not nil